internal/oidc/auth/auth_handler_test.go

```diff
@@ -54,58 +54,58 @@ func TestAuthorizationEndpoint(t *testing.T) {
 
 		fositePromptHasNoneAndOtherValueErrorQuery = map[string]string{
 			"error":             "invalid_request",
-			"error_description": "The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed\n\nParameter \"prompt\" was set to \"none\", but contains other values as well which is not allowed.",
-			"error_hint":        "Parameter \"prompt\" was set to \"none\", but contains other values as well which is not allowed.",
-			"state":             "some-state-value",
+			"error_description": "The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed\n\nUsed unknown value \"[none login]\" for prompt parameter",
+			"error_hint":        "Used unknown value \"[none login]\" for prompt parameter",
```

Comment:
We were using the prompt param test as a proxy for ensuring that we were calling into the appropriate corners of fosite to get all of the OIDC-related validations that we want. We should check to see if this config change means that an earlier validation is now triggering before we go down that code path, therefore preventing us from testing that we are really calling the thing that we wanted to call. If so, we can probably find a different OIDC validation to use to make sure we are using fosite correctly.

As Mo wrote on his TODO next to the `AllowedPromptValues` config, we should also figure out what is the best value for that config.